README.md

```diff
@@ -214,7 +235,7 @@ The planner is based on the [3DVFH+](http://ceur-ws.org/Vol-1319/morse14_paper_0
    or publish the DisparityImage as a simple sensor_msgs/Image
 
    ```bash
-   rosrun topic_tools transform /stereo/disparity /stereo/disparity_image sensor_msgs/Image 'm.image'
+   rosrun topic_tools transform /stereo/disparity /stereo/disparity_image sensor_msgs/Image 'm.image' 
```

Comment:
I tried to run this because it looked easier than working out what package to install. Got this error"
```
ubuntu@ubuntu:~/github/Firmware$ rosrun topic_tools transform /stereo/disparity /stereo/disparity_image sensor_msgs/Image 'm.image'
Unable to register with master node [http://localhost:11311]: master may not be running yet. Will keep trying.
```

Reply:
You didn't do `source setup/bash.rc` in the terminal where you run rosrun